### PR TITLE
Enhacements.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] Add config web params.
- [x] Add standart aliases.
- [x] Add standart params.
- [x] Adjust tests bundles.
- [x] Adjust github actions.
- [x] Add tests phan.
- [x] Add tests infection.
- [x] Update README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ It helps resolving dependencies and getting lists of files ready for generating
 [![Build Status](https://github.com/yiisoft/assets/workflows/build/badge.svg)](https://github.com/yiisoft/assets/actions)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/assets/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/assets/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/assets/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/assets/?branch=master)
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fassets%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/assets/master)
+[![static analysis](https://github.com/yiisoft/assets/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/assets/actions?query=workflow%3A%22static+analysis%22)
 
 
-## Intallation
+### Intallation
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md:21]: because this is how we have updated it in the other repositories.

[the remark this answers is not in the document]
Comment (samdark) [reply README.md:21]: Then we did that incorrectly in other places as well.

**Comment by samdark** [comment]
👍

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (schmunk42) [reply README.md]: @samdark @terabytesoftw There's an 's' missing.

[the remark this answers is not in the document]
Comment (samdark) [reply README.md]: Fixed. Thanks.

**Pull request merged**